game/player.py

```diff
@@ -1,20 +1,23 @@
 #!/usr/bin/env python
-from abc import ABCMeta, abstractmethod
-from game.gameboard import GameBoard
+from game.strategy_one import StrategyOne
 
 class Player(object):
-    __metaclass__ = ABCMeta
 
-    def __init__(self, color):
+    def __init__(self, color, strategy=None):
+        if strategy:
```

Comment:
Although in a lower note than the `Match` case, I'm not fully sold on having a default strategy.